mirrord/layer/tests/spawn.rs

```diff
@@ -23,48 +22,32 @@ pub use common::*;
 #[timeout(Duration::from_secs(60))]
 async fn node_spawn(dylib_path: &PathBuf) {
```

Comment:
I don't think this test verifies what it is supposed to, at least on MacOS. It used to verify that we are loaded into the newly spawned process by verifying it gets a new agent connection for it. Now, if we fail to load into a spawned process on macos, wouldn't `intproxy.try_recv` just get `None`, and we would break the loop and the app would exit successfully?
If that's the case the test should be updated to verify that `posix_spawn` works.
Either way, the documentation of this test should be updated.

Reply:
Ok ok, I'll look into it